malt/converters/slices.py

```diff
@@ -34,12 +34,21 @@ def _process_single_assignment(self, target, value):
     if not isinstance(target, gast.Subscript):
       return None
     s = target.slice
-    if isinstance(s, (gast.Tuple, gast.Slice)):
+    if isinstance(s, (gast.Tuple)):
       return None
 
     template = """
       target = ag__.set_item(target, key, item)
     """
+
+    if isinstance(s, (gast.Slice)):
+      template = template.replace("key", "slice(lower, upper)")
+      if s.upper is None:
```

Comment:
What if s.lower is None, or is that not possible?

Reply:
Yes it should be possible to handle it.